docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,77 @@
+# Development
+
+This document details how to get started contributing to the project.  This includes building and testing the core, templates and how to update them, and usage of the optional integration testing tooling.
+
+## Building
+
+To build the core project, run `make` from the repository root.  This will result in a `func` binary being generated.  Run `make help` for additional targets and `./func help` for usage-centric documentation.
+
+Building currently requires that `pkger` be available in your `$PATH`.  This can be installed with `go get github.com/markbates/pkger/cmd/pkger`.  See [Templates](#templates) below for more information on this dependency.
+
+To remove built artifacts, use `make clean`.
+
+## Testing
+
+To run core unit tests, use `make test`.
+
+## Linting
+
+Before submitting code in a Pull Request, please run `make check` and resolve any errors.  This creates and runs `bin/golangci-lint`.  For settings such as configured linters, see [.golangci.yaml](../.golangci.yaml).
+
+
+## Templates
+
+When a new Function is created, a few files are placed in the new Function's directory.  This includes source code illustrating a minimal Function of the requested type (language runtime and function signature) as well as Function metadata (`func.yaml`).
```

Comment:
```suggestion
When a new Function is created, a few files are placed in the new Function's directory.  This includes source code illustrating a minimal Function of the requested type (language runtime and function signature) as well as Function metadata in a `func.yaml` file.
```
Nit

Reply:
Thanks for the good suggestion.  My version was a bit terse and robotic.